Assets/Scripts/Systems/Modules/Module.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+
+namespace Systems.Modules
+{
+    [Serializable]
+    public class Module
+    {
+        public ModuleData Data;
+        public Coordinates PivotPosition;
```

Comment:
RootPosition makes more sense to me, but that might just be me

Reply:
YES much better